diff -urN aeolus-0.9.0/source/Makefile aeolus-0.9.0-patched/source/Makefile
--- aeolus-0.9.0/source/Makefile	2013-07-01 21:07:37.000000000 +1100
+++ aeolus-0.9.0-patched/source/Makefile	2014-02-11 00:27:19.445565464 +1100
@@ -18,14 +18,12 @@
 # -------------------------------------------------------------------------
 
 
-PREFIX = /usr/local
 SUFFIX := $(shell uname -m | sed -e 's/^unknown/$//' -e 's/^i.86/$//' -e 's/^x86_64/$/64/')
 LIBDIR = lib$(SUFFIX)
 VERSION = 0.9.0
 DISTDIR = aeolus-$(VERSION)
 CPPFLAGS += -DVERSION=\"$(VERSION)\" -DLIBDIR=\"$(PREFIX)/$(LIBDIR)\"
 CXXFLAGS += -O3 -Wall -MMD -MP
-CXXFLAGS += -march=native
 
 all:	aeolus aeolus_x11.so aeolus_txt.so
 
@@ -71,8 +69,6 @@
 	install -Dm 755 aeolus $(DESTDIR)$(PREFIX)/bin/aeolus
 	install -Dm 755 aeolus_x11.so $(DESTDIR)$(PREFIX)/$(LIBDIR)/aeolus_x11.so
 	install -Dm 755 aeolus_txt.so $(DESTDIR)$(PREFIX)/$(LIBDIR)/aeolus_txt.so
-	ldconfig $(PREFIX)/$(LIBDIR)
-
 
 clean:
 	/bin/rm -f *~ *.o *.d *.a *.so aeolus
